TF_Tooling_Web/src/Connection/MessageReceiver.ts

```diff
@@ -7,10 +7,9 @@ import {
     ServiceStatus,
     ServiceStatusCallback,
     TouchFreeRequest,
-    TrackingStateRequest,
     TrackingStateCallback,
     TrackingStateResponse,
-    WebSocketResponse
+    WebSocketResponse,
```

Comment:
This change seems out of place in this PR as it's the only TF_Tooling_Web change

Reply:
It's just a bit of clean up so I was thinking of just leaving this

Reply:
That's fine - just wanted to check it was intentional